packages/cloneit/build.sh

```diff
@@ -0,0 +1,30 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/alok8bb/cloneit
+TERMUX_PKG_DESCRIPTION="A cli tool to download specific GitHub directories or files"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_VERSION=1.0.0
+TERMUX_PKG_SRCURL=https://github.com/alok8bb/cloneit.git
+TERMUX_PKG_GIT_BRANCH="master"
```

Comment:
Having a fake version (1.0.0) and `TERMUX_PKG_GIT_BRANCH="master"` means that it will be hard to keep track of what commit the package was built from. We then can't know when to upgrade it, please change to using a date and commit hash, as in [for example the `gn` package](https://github.com/termux/termux-packages/blob/4f1a1bdc9304d93730314cbe47abd3b5e02ad868/packages/gn/build.sh#L5-L22)